packages/cyberstorm/src/components/Switch/Switch.tsx

```diff
@@ -8,16 +8,18 @@ export interface SwitchProps {
   state: boolean;
   onChange?: Dispatch<SetStateAction<boolean>>;
   disabled?: boolean;
+  Id?: string;
```

Comment:
The props shouldn't be capitalized, at least if their just strings. Use `id` instead.